src/azure-cli/azure/cli/command_modules/synapse/operations/sqlpoolsensitivitylabel.py

```diff
@@ -0,0 +1,152 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=unused-argument
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.mgmt.synapse.models import SensitivityLabel, SensitivityLabelSource
+from knack.util import CLIError
+
+
+def sqlpool_sensitivity_label_show(
+        client,
+        sql_pool_name,
+        workspace_name,
+        schema_name,
+        table_name,
+        column_name,
+        resource_group_name):
+
+    return client.get(
+        resource_group_name,
+        workspace_name,
+        sql_pool_name,
+        schema_name,
+        table_name,
+        column_name,
+        "current")  # ToDo: need to change to SensitivityLabelSource.current
+
+
+def sqlpool_sensitivity_label_update(
+        cmd,
+        client,
+        sql_pool_name,
+        workspace_name,
+        schema_name,
+        table_name,
+        column_name,
+        resource_group_name,
+        label_name=None,
+        information_type=None):
+    '''
+    Updates a sensitivity label. Custom update function to apply parameters to instance.
+    '''
+
+    # Get the information protection policy
+    from azure.mgmt.security import SecurityCenter
+    from msrestazure.azure_exceptions import CloudError
+
+    security_center_client = get_mgmt_service_client(cmd.cli_ctx, SecurityCenter, asc_location="centralus")
```

Comment:
>asc_location="centralus" [](start = 82, length = 24)

This hard-code location info is required?

Reply:
copied from sql db, I suggest to keep this, I don't know why sql db hard code `asc_location="centralus"`.

---
In reply to: [533177465](https://github.com/Azure/azure-cli/pull/16095#discussion_r533177465) [](ancestors = 533177465)